### ISSUE TITLE
Can't install master on a host with only SQLite installed

### ISSUE THREAD

**Issue by LukasReschke** [owncloud/core#16648]
1. Have a host with only the SQLite DB driver
2. Try to install ownCloud
3. :boom:

**Comment by LukasReschke** [comment]
Broken by @MorrisJobke with his change in setup.js at https://github.com/owncloud/core/commit/176b9674d29dc657a4b7000a9445fd7c3d74caee from https://github.com/owncloud/core/pull/16426

I have no idea about JS -> @MorrisJobke please take a look. THX.

**Comment by MorrisJobke** [comment]
Damn ... I thought I tested this with one DB only ... let me check that.

**Comment by MorrisJobke** [comment]
It's this call:

```
$('#selectDbType').buttonset('disable');
```

That results in 

```
Error: cannot call methods on button prior to initialization; attempted to call method 'option'
```

I will investigate further. It looks like the error didn't got triggered before, because it was an empty set. Now it sees the link as a button. -.-

if there is no buttonset available.

**Comment by MorrisJobke** [comment]
Fix is in #16650
